manifests/install_java.pp

```diff
@@ -63,12 +56,15 @@
     archive { "${tmp_dir}/aem.${part}":
       ensure  => present,
       source  => "${cert_base_url}/aem.${part}",
-      require => File[$tmp_dir],
-    } -> java_ks { "cqse-${idx}:/usr/java/default/jre/lib/security/cacerts":
-      ensure      => latest,
-      certificate => "${tmp_dir}/aem.${part}",
-      password    => 'changeit',
-      path        => ['/bin','/usr/bin'],
+      require => [
+        File["${tmp_dir}/java"],
+        Java::Download[$jdk_version],
+        ],
+      } -> java_ks { "cqse-${idx}:/usr/java/latest/jre/lib/security/cacerts":
```

Comment:
Is this path unique to a JDK version ? if yes please set a variable with the correct parameter depending on the JDK version Thanks